shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -570,6 +570,7 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
       weak_factory_(this),
       previous_route_id_(0),
       previous_routes_({}) {
+  [view_ retain];
```

Comment:
The UIView probably has a reference back to the VC too right? Wouldn't this produce a leak after a VC is dismissed? Since this object is probably never destroyed per engine.

Reply:
Yes, sorta.  The UIView would live until a new view controller is presented.  This isn't ideal so I'm working on an alternative PR right now.

Reply:
I've authored https://github.com/flutter/engine/pull/12232.  That will solve the original issue.  You know what though, we might still want this PR.  This one turns a crash into a leak and that one eliminates both.  This might be helpful to protect against future bugs.